weblate/templates/accounts/hosting.html

```diff
@@ -122,12 +122,14 @@ <h4 class="list-group-item-heading" id="billing-{{ billing.pk }}">
       <ul>
         <li>{% trans "Translated content has to be released under a license approved by OSI or recognized as libre by FSF." %}</li>
         <li>{% trans "All the source code has to be publicly available in a supported version control system." %}</li>
+        <li>{% trans "Translations have to be in the same repository, or under the same repository project/organization as the source code." %}</li>
         <li>{% trans "Using Weblate has been approved by the upstream project." %}</li>
         <li>{% trans "The project has had active development for at least three months." %}</li>
         <li>{% trans "You will have 14 days to set up your project." %}</li>
-        <li>{% trans "Please request project approval once project setup is completed." %}</li>
+        <li>{% trans "Please request project approval once you finnish the setup." %}</li>
         <li>{% trans "Unapproved projects will be removed." %}</li>
-        <li>{% trans "There is no guarantee of service availability, nor quality." %}</li>
+        <li>{% trans "There is no guarantee of service availability, nor quality for free service plan." %}</li>
+        <li>{% trans "Mention you use Weblate for translations in the README. It’ll help Weblate to be free for more projects and attract new translators." %}</li>
```

Comment:
This seems to raise some questions, see https://hosted.weblate.org/translate/weblate/application/en/?checksum=90955b5fe3f568a8#comments

Reply:
Thank you for the head’s up. I reworded it to make it clear. https://github.com/WeblateOrg/weblate/pull/4948